vmngclient/endpoints/device.py

```diff
@@ -0,0 +1,27 @@
+# mypy: disable-error-code="empty-body"
+from typing import List
+
+from pydantic import BaseModel
+
+from vmngclient.endpoints import APIEndpoints, post, versions
+
+
+class UnlockDeviceDetail(BaseModel):
+    deviceId: str
+    deviceIP: str
+
+
+class DeviceUnlockPayload(BaseModel):
+    deviceType: str
+    devices: List[UnlockDeviceDetail]
+
+
+class DeviceUnlockResponse(BaseModel):
+    parentTaskId: str
+
+
+class Device(APIEndpoints):
```

Comment:
`Device` class name may be too vague.
Because of enormous number of endpoints in vmanage. We tend to use autogenerated class names for endpoints to avoid name conflicts and ambiguity.
Resources could be find here: https://ghe-msite.cisco.com/sbasan/openapi-generator-vmanage

This specific endpoint could be found here: https://ghe-msite.cisco.com/sbasan/openapi-generator-vmanage/blob/master/2011/docs/ConfigurationDeviceInventoryApi.md
According to spec I suggest `ConfigurationDeviceInventory` and snake case module name: `configuration_device_inventory.py`